kolibri/plugins/style_guide/assets/src/views/content/button.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+
+  <div>
+    Button
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+  };
+
+</script>
+
+
+<style lang="stylus" scoped>
+
+  @require '~kolibri.styles.definitions'
```

Comment:
That's fair, but we should leave them out until/unless they're actually used to illustrate those properties. (Since these are `scoped` style sections, they have no effect outside the current style block.)